src/pages/tokenDrop/Landing.tsx

```diff
@@ -92,84 +105,96 @@ const Landing: FunctionComponent = () => {
         <Box width="100%">
           <SectionHeading>AVAILABLE SONG</SectionHeading>
 
-          <Box
-            sx={ {
-              display: "flex",
-              flexDirection: "row",
-              alignItems: "center",
-              justifyContent: "space-between",
-              width: "100%",
-              backgroundColor: theme.colors.grey600,
-              borderRadius: "6px",
-              overflow: "hidden",
-              p: 1.25,
-              pr: 2,
-            } }
-          >
-            <Stack spacing={ 2 } direction="row" alignItems="center">
-              <Box
-                display="flex"
-                height={ 2 }
-                width={ 2 }
-                margin={ 2 }
-                justifyContent="center"
-                alignItems="center"
-              >
-                { isPlaying ? (
-                  <IconButton
-                    aria-label="stop song"
-                    onClick={ () => audio.stop() }
-                  >
-                    <StopIcon />
-                  </IconButton>
-                ) : (
-                  <IconButton
-                    aria-label="play song"
-                    onClick={ () => audio.play() }
-                  >
-                    <PlayIcon />
-                  </IconButton>
-                ) }
-              </Box>
-
-              <img
-                src={ artistAssets.albumArtXs }
-                style={ { width: 40, height: 40, borderRadius: "50%" } }
-                alt="artist profile"
-              />
-
-              <Box>
-                <Typography variant="h4" fontWeight={ 700 }>
-                  { projectDetails.songName }
-                </Typography>
-                <Typography variant="subtitle2">
-                  { projectDetails.artistName }
-                </Typography>
-              </Box>
-            </Stack>
+          <Box display="flex" alignItems="center">
```

Comment:
I added this box as a wrapper to be able to add a new tooltip in this section idk why git thinks i modified a lot here